askama_derive/src/generator.rs

```diff
@@ -873,7 +873,14 @@ impl<'a> Generator<'a> {
         };
 
         let mut expr_buf = Buffer::new(0);
+        let borrow_val = !is_copyable(val);
+        if borrow_val {
+            expr_buf.write("&(");
```

Comment:
Would it be worth it to not generate the parens if the expression is a literal or a field?

Reply:
I don't think it'd be worth it. This isn't done in `Generator::_visit_args`

Reply:
Also literals are explicitly rejected in `is_copyable`.